examples/js/loaders/AssimpJSONLoader.js

```diff
@@ -163,9 +157,9 @@ THREE.AssimpJSONLoader.prototype = {
 					b = face.b * 4;
 					c = face.c * 4;
 					face.vertexColors = [
-						makeColor( a ),
-						makeColor( b ),
-						makeColor( c )
+						color.fromArray( a ),
+						color.fromArray( b ),
+						color.fromArray( c )
```

Comment:
@mrdoob  This looks better to me, but it is a guess:

    new THREE.Color().fromArray( in_color[ a ] )

Is `json.colors` in RGBA format?